x-pack/filebeat/module/suricata/eve/test/eve-alerts.log-expected.json

```diff
@@ -53,7 +53,6 @@
         "suricata.eve.flow.bytes_toserver": 347,
         "suricata.eve.flow.pkts_toclient": 3,
         "suricata.eve.flow.pkts_toserver": 4,
-        "suricata.eve.flow.start": "2018-10-03T14:42:44.613469+0000",
```

Comment:
I haven't removed these fields from `fields.yml` in case we want to reserve them to be used in the future. Happy to remove them in any case if we decide to go on this way.